README.md

```diff
@@ -1,6 +1,14 @@
+# prometheus-test-environment
+A Kubernetes cluster preconfigured for E2E testing and benchmarking Prometheus
+
+The design details and instructions on how to run can be found [here](design.md)
+
 # A benchmarking tool preconfigured for testing Prometheus
 1. Create the k8s cluster described by a configuration file.
 2. Apply a k8s resource file to create all required pods,services and config maps.
+3. Create a service account with Kubernetes Admin access 
+4. Follow [this](https://github.com/coreos/prometheus-operator/issues/357#issuecomment-363498455) to get primaryemail of the serviceaccount user and add this to \_rbac.yaml
+
```

Comment:
do we still need this step?

Reply:
The ReadME is outdated. I'll update it

Reply:
But we still need a service-account to authenticate the GKE API